src/Math/include/BipedalLocomotion/Math/CARE.h

```diff
@@ -0,0 +1,94 @@
+/**
+ * @file CARE.h
+ * @authors Giulio Romualdi
+ * @copyright 2020 Istituto Italiano di Tecnologia (IIT). This software may be modified and
+ * distributed under the terms of the GNU Lesser General Public License v2.1 or any later version.
+ */
+
+#ifndef BIPEDAL_LOCOMOTION_MATH_CARE_H
+#define BIPEDAL_LOCOMOTION_MATH_CARE_H
+
+#include <Eigen/Dense>
+
+#include <memory>
+
+#include <BipedalLocomotion/ParametersHandler/IParametersHandler.h>
+
+namespace BipedalLocomotion
+{
+namespace Math
+{
+
+/**
+ * Computes the unique stabilizing solution S to the continuous-time algebraic equation
+ * @f[
+ *   S A + A' S - S B R^{-1} B' S + Q = 0
+ * @f]
+ * @note This class implements the algorithm presented in the paper: Solving the algebraic
+ * Riccati equation with the matrix sign function
+ * https://www.sciencedirect.com/science/article/pii/0024379587902229
+ */
+class CARE
+{
+    /**
+     * Private implementation
+     */
+    struct Impl;
+    std::unique_ptr<Impl> m_pimpl;
+
+public:
+
+    /**
+     * Initialize the continuous algebraic riccati equation solver.
+     * @param handler pointer to the parameter handler.
+     * @note The following parameters are required by the class:
```

Comment:
this note needs to be changed since the parameters are optional, rather, the method itself is optional.

Reply:
Fixed in https://github.com/dic-iit/bipedal-locomotion-framework/commit/3c6176db6cd43d8d0b98d3c0c2deec7c928e3393